src/Ethereum/Transaction.cpp

```diff
@@ -58,3 +63,16 @@ Data Transaction::buildERC721TransferFromCall(const Data& from, const Data& to,
     func.encode(payload);
     return payload;
 }
+
+Data Transaction::buildERC1155TransferFromCall(const Data& from, const Data& to, uint256_t tokenId, uint256_t value, const Data& data) {
```

Comment:
```
@param _from    Source address
        @param _to      Target address
        @param _ids     IDs of each token type (order and length must match _values array)
        @param _values  Transfer amounts per token type (order and length must match _ids array)
        @param _data    Additional data with no specified format, MUST be sent unaltered in call to the `ERC1155TokenReceiver` hook(s) on `_to`
    */
```

Should we support multiple `Ids` and `values` and values as per spec: https://eips.ethereum.org/EIPS/eip-1155

Reply:
that's another call `safeBatchTransferFrom`, we can add this later if we need to support transfer multiple collectibles in the app